README.md

```diff
@@ -49,8 +49,8 @@ clientSettings:
 ```
 
 * `clientSettings.concurrency` is an optional parameter that shows the concurrency of Nebula Graph Client, i.e. the connection number of Nebula Graph Server, the default value is 10.
```

Comment:
`clientSettings.concurrency` is an optional parameter that shows the concurrency of Nebula Graph Client, i.e. the connection number of Nebula Graph Server.  The default value is 10.